package.json

```diff
@@ -7,10 +7,46 @@
     "url": "https://github.com/oneblink/cra-template-oneblink-forms-app/issues"
   },
   "dependencies": {
-    "@blinkmobile/geolocation": "^2.0.3",
-    "@oneblink/apps": "^0.10.0-beta.10",
-    "@oneblink/apps-react": "^0.4.10-beta.1",
-    "@oneblink/sdk": "0.9.5"
+    "@oneblink/apps": "^0.10.1",
+    "@oneblink/apps-react": "^0.4.11",
+    "@oneblink/sdk-core": "^0.1.0",
+    "@styled-icons/material": "^10.34.0",
+    "@styled-icons/material-outlined": "^10.34.0",
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^11.2.7",
```

Comment:
the only ones that should be in here are the oneblink ones as we will eventually setup a dependabot + gitlab actions to auto update the `template.json` package

Reply:
any new dependencies should be put into `template.json`

Reply:
oh I seeeeeeeeeeeee